### DISCUSSION TITLE
Flutter specific Discussions of freeRASP community

### DISCUSSION THREAD

**Discussion by syakymchuk** [talsec/Free-RASP-Flutter#36] [Announcements]
## 👋 Welcome to freeRASP for the Flutter community!
  
Here you can find only Flutter-specific topics.

All the freerRASP platform agnostic topics will be migrated to freeRASP [Community Discussion](https://github.com/talsec/Free-RASP-Community/discussions) here: 

https://github.com/talsec/Free-RASP-Community/discussions

**Comment by venzeo** [comment]
Thank you in the first place for supporting open source community with your FreeRASP project. 
Question:
1. How to restrict the plugin to send any data to your or any 3rd party server? If contact email is not set, then no data are being populated?

**Reply by syakymchuk** [reply]
Hello @venzeo, 

Thank you for trying freeRASP.

You can't do this in freeRASP subscription. This is a limitation of freeRASP security monitoring and the concept of one binary SDK for all users . Such configuration is possible only in commercial plans. We are introducing the low-cost option **lightRASP**. You can see some details and put further questions in this [thread](https://github.com/orgs/talsec/discussions/8). 

Make sure the level of features and security of lightRASP/freeRASP is enough for the value of your App. See comparison [here](https://github.com/orgs/talsec/discussions/8)   

Have a great day!
